disco/types/channel.py

```diff
@@ -120,27 +120,31 @@ class Channel(SlottedModel, Permissible):
         Channel permissions overwrites.
     """
     id = Field(snowflake)
+    type = Field(enum(ChannelType))
     guild_id = Field(snowflake)
+    position = Field(int)
+    permission_overwrites = AutoDictField(PermissionOverwrite, 'id', alias='permission_overwrites')
```

Comment:
I personally think that it should have remained "overwrites" for easier access. I get it's for staying updated with the API Documentation, but it's not hard to tell it's for permissions with the Disco documentation, as well as it showing it's a dict full of PermissionOverwrites